src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,210 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
 
+    public static final Scanner CONSOLE_READER = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("\nWrite a command: ");
+        String textFromConsole = CONSOLE_READER.nextLine();
+        while (!textFromConsole.equals("exit")) {
+            if (textFromConsole.equals("help")) {
+                help();
+            } else {
+                if (myPattern("^help+\\s(\\[+)\\w+\\]$",
+                        textFromConsole).find()) {
+                    String command = textFromConsole
+                            .replaceAll("\\w+\\s\\[", "")
+                            .replace("]", "");
+                    help(command);
+                } else if (myPattern("^\\w+\\s(\\[+)\\w+\\]\\s(\\[)\\w+\\.+\\w+\\]$",
+                        textFromConsole).find()) {
+                    String[] commandDates = textFromConsole
+                            .replaceAll("[^a-z\\s.]", "")
+                            .split("\\s");
+                    switch (commandDates[0]) {
+                        case "read":
+                            read(commandDates[1], commandDates[2]);
+                            break;
+                        case "create":
+                            create(commandDates[1], commandDates[2]);
+                            break;
+                        case "info":
+                            info(commandDates[1], commandDates[2]);
+                            break;
```

Comment:
make another case for  "help" and default for esle block from 49 line(move it to separate method)